task/rhacs-deployment-check/3.71/README.md

```diff
@@ -2,48 +2,101 @@
 
 Check a deployment manifest against RHACS deploy lifecycle policies to validate a pipeline run using `roxctl`.
 
+**Note: this Task is not backwards compatible with the `3.71` versions as it changes the parameters and token configuration.**
+
 ## Prerequisites
 
-This task requires an active installation of [Red Hat Advanced Cluster Security (RHACS)](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet).  It also requires configuration of secrets for the Central endpoint and an API token with at least CI privileges.
+This task requires an active installation of [Red Hat Advanced Cluster Security (RHACS)](https://www.redhat.com/en/resources/advanced-cluster-security-for-kubernetes-datasheet) or [StackRox](https://www.stackrox.io).  It also requires configuration of an authorization token with at least CI privileges.
 
 <https://www.redhat.com/en/technologies/cloud-computing/openshift/advanced-cluster-security-kubernetes>
 
 ## Install the Task
 
 ```bash
-kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/rhacs-deployment-check/3.71/raw
+kubectl apply -f https://api.hub.tekton.dev/v1/resource/tekton/task/rhacs-deployment-check/4.0/raw
 ```
 
 ## Parameters
 
-- **`deployment`**: Filename of deployment manifest. May be relative to workspace root or fully qualified. (example -- kustomize/overlays/dev/deployment.yaml)
-- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central endpoint and registry. Examples: _"true", **"false"**_.
+- **`deployment`**: Filename of deployment manifest. May be relative to source workspace root or fully qualified. (example -- kustomize/overlays/dev/deployment.yaml)
 - **`output_format`**:  Examples: _**table**, csv, json, junit_
-- **`rox_central_endpoint`**: Secret containing the address:port tuple for StackRox Central. Default: _**rox-central-endpoint**_
-- **`rox_api_token`**: Secret containing the StackRox API token with CI permissions. Default: _**rox-api-token**_
+
+- **`rox_central_endpoint`**: The address:port tuple for StackRox Central. Default: **central.stackrox.svc:443**
+- **`insecure-skip-tls-verify`**: Skip verification the TLS certs of the Central endpoint and registry. Examples: _"true", **"false"**_.
+- **`rox_config_dir`**: Path to the roxctl config directory within the `roxctl-config` workspace (if machine to machine authentication is used). Mutually exclusive with **`rox_token_file`**. The path should be prefixed with `/roxctl-config`. Examples: _"/roxctl-config", **""**_.
+- **`rox_token_file`**: Path to the API Token file (if API Token authentication is used). Mutually exclusive with **`rox_config_dir`**. The path should be prefixed with `/rox-api-token-auth`. Examples: _**""**, "/rox-api-token-auth/rox_api_token"_.
+- `rox_image`: The image providing the roxctl tool (optional). Default: quay.io/stackrox-io/roxctl:4.4.2 (this is also the minimum version working with this task). 
```

Comment:
Would be great to make this more prominent like we did with the breaking change at the top.